### PR TITLE
[glide] Update m3db and m3x dependencies

### PR DESCRIPTION
**Pull request by jeromefroe**
This PR updates the m3db and m3x dependencies for `m3nsch`.

@prateek once https://github.com/m3db/m3db/pull/353 lands I'll update the version of `m3db` in Glide to reflect master.

### CODE REVIEW

File: m3nsch_server/config/config.go [hunk 1]
```diff
@@ -21,16 +21,16 @@
...
 )
 
 // Configuration represents the knobs available to configure a m3nsch_server
 type Configuration struct {
-	Server  ServerConfiguration  `yaml:"server"`
-	M3nsch  M3nschConfiguration  `yaml:"m3nsch" validate:"nonzero"`
-	Metrics MetricsConfiguration `yaml:"metrics" validate:"nonzero"`
+	Server  ServerConfiguration             `yaml:"server"`
+	M3nsch  M3nschConfiguration             `yaml:"m3nsch" validate:"nonzero"`
+	Metrics instrument.MetricsConfiguration `yaml:"metrics" validate:"nonzero"`
```
Comment (prateek) [inline m3nsch_server/config/config.go:34]: Nice!

File: m3nsch_server/config/config.go [hunk 2]
```diff
@@ -21,16 +21,16 @@
 package config
 
 import (
+	"github.com/m3db/m3x/instrument"
```
Comment (prateek) [inline m3nsch_server/config/config.go:24]: nit: import ordering

Comment (jeromefroe) [reply m3nsch_server/config/config.go:24]: good catch, will fix

**Review by prateek — APPROVED** [review]
LGTM with one nit

**Pull request merged**